numba/tests/test_unicode.py

```diff
@@ -904,6 +908,40 @@ def try_compile_bad_optional(*args):
                                      "'%s'.%s('%s')?" % (string, case_name,
                                                          chars))
 
+    def test_istitle(self):
+        pyfunc = istitle_usecase
+        cfunc = njit(pyfunc)
+        error_msg = "'{0}'.py_istitle() = {1}\n'{0}'.c_istitle() = {2}"
+        
+        unicode_title = [x.title() for x in UNICODE_EXAMPLES]
+        special = [
+            '',
+            '    ',
+            '  AA  ',
+            '  Ab  ',
+            '1',
+            'A123',
+            'A12Bcd',
+            '+abA',
+            '12Abc',
+            'A12abc',
+            '%^Abc 5 $% Def'
+            '𐐁𐐩',
+            '𐐧𐑎',
+            '𐐩',
+            '𐑎',
+            '🐍 Is',
+            '🐍 NOT',
+            '👯Is'
+        ]
+        ISTITLE_EXAMPLES = UNICODE_EXAMPLES + unicode_title + special
```

Comment:
Please consider CPython test parameters, especially symbol `\u1FFc`:
https://github.com/python/cpython/blob/201c8f79450628241574fba940e08107178dc3a5/Lib/test/test_unicode.py#L604-L605 
This symbol is not covered by this implementation.

Reply:
Will add this, thanks!